packages/next/src/lib/constants.ts

```diff
@@ -137,6 +137,18 @@ const WEBPACK_LAYERS_NAMES = {
    * The browser client bundle layer for App directory.
    */
   appPagesBrowser: 'app-pages-browser',
+  /**
+   * The browser client bundle layer for Pages directory.
+   */
+  pagesDirBrowser: 'pages-dir-browser',
+  /**
+   * The Edge Lite bundle layer for Pages directory.
+   */
+  pagesDirEdge: 'pages-dir-edge',
+  /**
+   * The Node.js bundle layer for Pages directory.
+   */
+  pagesDirNode: 'pages-dir-node',
```

Comment:
IMO we should remove `pagesDirNode` for now as we don't really bundle all for pages router nodejs side, it could get lead to confusion. also in the future we might want to bundle gssp separately, it's not just nodejs runtime

Reply:
I'm not following. What is the problem this creates?